src/plot_api/plot_api.js

```diff
@@ -326,6 +326,9 @@ Plotly.plot = function(gd, data, layout, config) {
     // so that the caller doesn't care which route we took
     return Promise.all(gd._promises).then(function() {
         return gd;
+    }, function() {
+        // clear the promise queue if one of them got rejected
+        gd._promises = [];
```

Comment:
Yep, I'm fine with logging something.

Why it this _dangerous_ though?

To add more info: I added :arrow_double_up: to ensure that a `Plotly.relayout` call that results in a rejected error can be followed by another `Plotly.relayout` without any extra work. Maybe the line :arrow_double_up: belongs in `Plotly.relayout` and in `Plotly.restyle`. Thoughts?